src/bp-xprofile/classes/class-bp-xprofile-user-admin.php

```diff
@@ -38,7 +36,7 @@ public static function register_xprofile_user_admin() {
 			$bp->profile->admin = new self;
 		}
 
-		return $bp->profile->admin;
+		$bp->profile->admin;
```

Comment:
Once again, I'd like to test this change to make sure it doesn't break the xProfile WP Admin screen.

Reply:
Just tested this. I saw no side effects. It's could to go.